app/game-session.coffee

```diff
@@ -1,6 +1,6 @@
 {nx} = require 'nexus-node'
 
-RoundPhase    = require 'cssqd-shared/models/round-phase'
+RoundPhase    = require '../shared/models/round-phase'
```

Comment:
it didn't work for me for some reason... does it really work like webpack aliases?
summoning @kirilknysh